Formula/jsonnet-bundler.rb

```diff
@@ -0,0 +1,24 @@
+class JsonnetBundler < Formula
+  desc "Package manager for Jsonnet"
+  homepage "https://github.com/jsonnet-bundler/jsonnet-bundler"
+  url "https://github.com/jsonnet-bundler/jsonnet-bundler.git",
+    :tag      => "v0.4.0",
+    :revision => "447344d5a038562d320a3f0dca052611ade29280"
+  license "Apache-2.0"
+
+  depends_on "go@1.12" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GO111MODULES"] = "on"
```

Comment:
if `go.mod` present, this would be auto-on.
```suggestion
```

Reply:
Resolved in push.